modules/candidate_list/test/candidate_listTest.php

```diff
@@ -26,12 +26,11 @@ class CandidateListTestIntegrationTest extends LorisIntegrationTestWithCandidate
     //filter location
     static $PSCID          = ".col-xs-12:nth-child(2) > .row .form-control";
     static $DCCID          = ".col-xs-12:nth-child(3) > .row .form-control";
-    static $visitLabel     = ".col-xs-12:nth-child(4) .form-control";
-    static $site           = ".col-xs-12:nth-child(5) .form-control, select";
-    static $entityType     = ".col-xs-12:nth-child(7) .form-control, select";
-    static $sex            = "#candidateList_filter > div > div > fieldset >".
-                             " div:nth-child(11) > div > div > select";
-    static $project        = ".col-xs-12:nth-child(15) .form-control, select";
+    static $visitLabel     = "select[name=\"visitLabel\"]";
```

Comment:
you're not interpolating anything, why don't you just change the quoting style?

ie `static $visitLabel = 'select[name="visitLabel"]';`

Reply:
because i copy pasted and my IDE did the escaping